dev-requirements.txt

```diff
@@ -5,6 +5,7 @@ sphinx
 sphinx_rtd_theme
 sphinx-autobuild
 nose
+codecov
```

Comment:
Moved this to dev-requirements so it is available in the Travis top-level script